tealer/utils/analyses.py

```diff
@@ -113,7 +114,9 @@ def detect_missing_txn_check(
         if isinstance(ins, Return):
             if len(ins.prev) == 1:
                 prev = ins.prev[0]
-                if isinstance(prev, Int) and prev.value == 0:
+                if (isinstance(prev, Int) and prev.value == 0) or (
+                    isinstance(prev, PushInt) and prev.value == 0
+                ):
```

Comment:
```suggestion
                if (isinstance(prev, (Int, PushInt)) and prev.value == 0):
```
I think you can get away with something like this and simplify the expressions

Reply:
Yep, It does simplify the expression. Unfortunately, mypy errors when we do that. mainly, `error: Too many arguments for "isinstance"` and `"Instruction" has no attribute "value"`. (Instruction is the super class of `Int` and `PushInt`)